README.md

```diff
@@ -1,97 +1,51 @@
-# snapcraft.io
+# ![snapcraft.io](https://assets.ubuntu.com/v1/f46fbbb9-snapcraft-logo--web.png "Snapcraft").io codebase
 
 [![CircleCI build status](https://circleci.com/gh/canonical-web-and-design/snapcraft.io.svg?style=shield)](https://circleci.com/gh/canonical-web-and-design/snapcraft.io) [![Code coverage](https://codecov.io/gh/canonical-web-and-design/snapcraft.io/branch/master/graph/badge.svg)](https://codecov.io/gh/canonical-web-and-design/snapcraft.io)
 
-This is the application for [snapcraft.io](https://snapcraft.io) website.
+Snaps are applications packaged with all their dependencies to run on all popular Linux distributions from a single build. They update automatically and roll back gracefully. This repo is the application for the [snapcraft.io](https://snapcraft.io) website.
 
-## Local development
-
-The simplest way to run the site locally is to first [install Docker](https://docs.docker.com/engine/installation/) (on Linux you may need to [add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)), and then use the `./run` script:
-
-``` bash
-./run
-```
-
-Once the containers are setup, you can visit <http://127.0.0.1:8004> in your browser.
-
-### Use staging APIs
-
-To use staging APIs locally you can add the following lines to an `.env.local` file:
-
-```bash
-SNAPCRAFT_IO_API=https://api.staging.snapcraft.io/api/v1/
-SNAPCRAFT_IO_API_V2=https://api.staging.snapcraft.io/v2/
-DASHBOARD_API=https://dashboard.staging.snapcraft.io/dev/api/
-DASHBOARD_API_V2=https://dashboard.staging.snapcraft.io/api/v2/
-LOGIN_URL=https://login.staging.ubuntu.com
-```
-
-### Using Sentry error tracker
-
-For development purposes, visit https://sentry.io/signup/, signup and setup a project. By then you will have a sentry DSN string like:
-
-```
-https://<user>:<secret>@sentry.io/<project_id>
-```
-
-Create or update you `.env.local` file:
-
-```
-SENTRY_DSN=<DSN_FROM_ABOVE>
-```
+If you are interested in Snaps, Snapping and Snapcraft, there is an active [discourse forum](https://forum.snapcraft.io/) that we encourage developers to join.
 
-The application will be reporting errors to your `sentry.io` project from now on.
+The site is largely maintained by the [Web and Design team](https://ubuntu.com/blog/topics/design) at [Canonical](https://www.canonical.com). It is a simple, stateless website project based on [Flask](https://flask.palletsprojects.com/en/1.1.x/) and hosted on a [Charmed Kubernetes](https://ubuntu.com/kubernetes) cluster.
```

Comment:
"simple" 🤣

Reply:
I might have borrowed that line from ubuntu.com